### DISCUSSION TITLE
Suggestion: Add repo description to watchlist email notifications

### DISCUSSION THREAD

**Discussion by kirklee** [community/community#55205] [Repositories]
### Select Topic Area

Product Feedback

### Body

I watch many repos and get email updates for them. But the emails only show the changes and the repo name, not what the repo is about. 

Because I'm old, forgetful, or all three, I often forget what the repo does and have to click through to see the README. It would be nice if the emails also included the first line of the README as a reminder. For example, for juanfont/headscale v0.22.2, I would see that it is “An open source, self-hosted implementation of the Tailscale control server”. This would make the emails more useful for me. Thanks.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @kirklee! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
